### PR TITLE
Add keep truthy overload to bitwise not

### LINKED ISSUE #894: Add keep if true monograph
A monograph version of `~ḃ`

### PR DESCRIPTION
**Pull request by Seggan**
Resolves #894

### CODE REVIEW

**Review by Steffan153 — APPROVED** [review]

File: documents/knowledge/elements.yaml [hunk 1]
```diff
@@ -2405,10 +2405,12 @@
   overloads:
       num: ~a
       str: any_upper(a)
+      lst: filter(lst, is_true)
   vectorise: true
```
Comment (Steffan153) [inline documents/knowledge/elements.yaml:2409]: You need to set vectorise to false here now

**Review by Steffan153 — CHANGES_REQUESTED** [review]

**Review by Steffan153 — APPROVED** [review]

File: documents/knowledge/elements.yaml [hunk 2]
```diff
@@ -2405,10 +2405,12 @@
   overloads:
       num: ~a
       str: any_upper(a)
-  vectorise: true
+      lst: filter(lst, is_true)
```
Comment (cgccuser) [inline documents/knowledge/elements.yaml:2408]:
```suggestion
      lst: keep truthy
```
I don't think there's much point in writing code if it's not the actual code we use. Just an English phrase should be enough

File: documents/knowledge/elements.yaml [hunk 3]
```diff
@@ -2405,10 +2405,12 @@
   overloads:
```
Comment (cgccuser) [inline documents/knowledge/elements.yaml:2405]: Could you update the description too?

**Review by cgccuser — APPROVED** [review]
Looks good to me, couple minor things

**Pull request merged**